commands/build.go

```diff
@@ -43,6 +45,7 @@ func init() {
 	buildCmd.Flags().StringArrayVarP(&buildArgs, "build-arg", "b", []string{}, "Add a build-arg for Docker (KEY=VALUE)")
 	buildCmd.Flags().StringArrayVarP(&buildOptions, "build-option", "o", []string{}, "Set a build option, e.g. dev")
 	buildCmd.Flags().StringVar(&tag, "tag", "", "Override latest tag on function Docker image, takes 'sha' or 'branch'")
+	buildCmd.Flags().StringArrayVar(&buildLabels, "build-label", []string{}, "Add a label for Docker image (LABEL=VALUE)")
```

Comment:
Is there a reason we can't use `--label` here? Or at least `--image-label` - it looks like `build-` is a copy paste from the build args and doesn't really communicate the purpose of this flag.

Reply:
Good question. It may have been my suggestion.

`--label` is used for deployments and might conflict when used in `faas-cli up`

WDYT?

Reply:
In think we should use `--image-label` or `--build-label`, `--label` will have conflict with `--label` flag for deployment.

Reply:
The contexts are different.

Reply:
Nvm, up spans both.

Reply:
`--image-label` might be more correct. I went for `--build-label`.